test/fdspaces.jl

```diff
@@ -567,3 +567,63 @@ end
     @test conv_div_cos_f[1] ≤ conv_div_cos_f[2] ≤ conv_div_cos_f[3]
 
 end
+
+@testset "Biased interpolation" begin
```

Comment:
Is the term "biased" very common to express one-sided (forward/backward) operators in this community? It's not that common in the applied math / scientific computing communities where biases are generally left for statics and ML contexts

Reply:
I was just following the existing naming convention. I'm also used to using one-sided/forward/backward operators.